internal/storage/v1/cassandra/spanstore/dbmodel/model.go

```diff
@@ -15,11 +15,13 @@ const (
 	childOf     = "child-of"
 	followsFrom = "follows-from"
 
-	stringType  = "string"
-	boolType    = "bool"
-	int64Type   = "int64"
-	float64Type = "float64"
-	binaryType  = "binary"
+	stringType   = "string"
+	boolType     = "bool"
+	int64Type    = "int64"
+	float64Type  = "float64"
+	binaryType   = "binary"
+	spanKindKey  = "span.kind"
+	FirehoseFlag = int32(8)
```

Comment:
this is not the same kind of constant, why group it with others?
```suggestion

	FirehoseFlag = int32(8)
```